broker/src/main/java/io/camunda/zeebe/broker/system/partitions/ZeebePartitionHealth.java

```diff
@@ -23,17 +26,22 @@ class ZeebePartitionHealth implements HealthMonitorable {
   private final String name;
   private final Set<FailureListener> failureListeners = new HashSet<>();
   private HealthReport healthReport = HealthReport.unhealthy(this).withMessage("Initial state");
+  private final PartitionTransitionContext partitionContext;
   /*
   Multiple factors determine ZeebePartition's health :
   * - servicesInstalled: indicates if role transition was successful and all services are installed
   * - diskSpaceAvailable
+  * - if the partition is not blocked in a transition step
   */
   private boolean servicesInstalled;
   // We assume disk space is available until otherwise notified
   private boolean diskSpaceAvailable = true;
+  private final long timeoutInMilliseconds = 60_000L;
```

Comment:
:wrench: Needs a more descriptive name.
:wrench: IMO we should almost always use `Duration` instead of plain numbers.